js/index.js

```diff
@@ -2,17 +2,19 @@ define([
   'core/js/adapt',
   './adapt-contrib-xapi',
   './adapt-offlineStorage-xapi'
-], function(Adapt) {
+], function(Adapt, xapi, offlineStorage) {
 
   Adapt.on('app:dataLoaded', initialise);
```

Comment:
offlineStorage:prepare might be a better event, if you look at the ConfigModel you'll see there's a special event for initializing the offlineStorage handler.

Reply:
Triggering on this actually leaves the initialization hanging due to the order of events triggered I believe

Reply:
Weird. Good to know though. Wish I had more time to take a proper look.